lib/core/MongoDbConfirmationStore.ts

```diff
@@ -0,0 +1,45 @@
+import IConfirmationStore from './interfaces/IConfirmationStore';
+import MongoDbStore from '../common/MongoDbStore';
+
+interface ConfirmationModel {
+  anchorString: string;
+  submittedAt: number;
+  confirmedAt: number | undefined;
+}
+
+/**
+ * Implementation of LastWriteStore that stores the last update per write
+ */
+export default class MongoDbConfirmationStore extends MongoDbStore implements IConfirmationStore {
```

Comment:
Should we add index for both `submittedAt` and `anchorString`?

Reply:
Yes